src/sentry/runner/commands/run.py

```diff
@@ -253,6 +253,61 @@ def taskworker(rpc_host: str, max_task_count: int, **options: Any) -> None:
         raise SystemExit(exitcode)
 
 
+@run.command()
+@log_options()
+@configuration
+@click.option(
+    "--repeat",
+    type=int,
+    help="Number of messages to send to the kafka topic",
+    default=100,
+    show_default=True,
+)
+@click.option(
+    "--kwargs",
+    type=str,
+    help="Task function keyword arguments",
+)
+@click.option(
+    "--args",
+    type=str,
+    help="Task function arguments",
+)
+@click.option(
+    "--task-function",
+    type=str,
+    help="The function name of the task to execute located in the module",
+    required=True,
+)
+@click.option(
+    "--path",
+    type=str,
+    help="The path to the task module",
+    required=True,
+)
```

Comment:
We could probably combine these and have a positional argument that takes the module path to the task eg. `sentry run taskbroker_send_tasks sentry.issues.tasks.do_things`. The `import_string()` utility function can help with resolving the module path to the function.